app/controllers/people_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PeopleController < ApplicationController
+
+def index
+  @header = "Minions"
+  @all_people = Person.all
+end
+
+def show
+  @header = "Person Details"
+  @person = Person.find(params[:id])
+end
+
+def tasks
+  @person = Person.find(params[:id])
+  @tasks = Task.where(:person_id => @person.id)
```

Comment:
You should also be able to use `@person.tasks` to retrieve the associated tasks.